perf/lua/uri_escape_unescape.lua

```diff
@@ -1,9 +1,19 @@
--- It is recommended to run benchmark using taskset for stable results.
--- taskset -c 1 tarantool uri_escape_unescape.lua
-
 local uri = require("uri")
 local clock = require("clock")
 local t = require("tarantool")
+local benchmark = require("benchmark")
+
+local USAGE = [[
+
+ Usage: taskset -c 0x1 tarantool uri_escape_unescape.lua
```

Comment:
`Usage: taskset -c 0x1 tarantool uri_escape_unescape.lua` -> ` Usage: tarantool uri_escape_unescape.lua [options]`.

Reply:
I moved it to the generated part without mentioning the custom taskset directly.

Reply:
Thanks!